auctioneer/client.go

```diff
@@ -1100,13 +1121,15 @@ func (c *Client) readIncomingStream() { // nolint:gocyclo
 			select {
 			case c.FromServerChan <- msg:
 			case <-c.quit:
+				return
```

Comment:
Hmm, I wonder if there's some sort of static check library that can help us spot these in the future...